flixel/text/FlxInputText.hx

```diff
@@ -1055,7 +1061,7 @@ class FlxInputText extends FlxText implements IFlxInputText
 				{
 					restartCaretTimer();
 				}
-				onTextChange.dispatch(text, ENTER_ACTION);
+				onEnter.dispatch(text);
```

Comment:
why wouldn't people want text change events to include enters?

Reply:
Because its dispatched even if the text isn't multiline and thus doesn't create a new line, so it's not really a "change" event. If the text is multiline, then the new line gets dispatched in `INPUT_ACTION`.

Reply:
Thats a decent argument for why people wouldn't want enters to dispatch text change events each time, but I still think hitting enter in a multiline text should dispatch a text change event

Reply:
> I still think hitting enter in a multiline text should dispatch a text change event

Yes, thats why I said the signal still gets dispatched with `INPUT_ACTION` whenever you press enter to add a new line, as the same `addText` function is called for when you input text normally.

Reply:
tried this for a reference, it doesn't dispatch when multiline if false, but does when true. which makes sense
```hx
text = new TextField();
text.text = "The quick brown fox jumps over the lazy dog";
text.type = INPUT;
text.multiline = false;
text.addEventListener(openfl.events.Event.CHANGE, (e)->trace(text.text));
```

Reply:
> then the new line gets dispatched in `INPUT_ACTION`.

Wait are you saying you took out ENTER_ACTION becuase it's covered by INPUT_ACTION?

Reply:
> > then the new line gets dispatched in `INPUT_ACTION`.
> 
> Wait are you saying you took out ENTER_ACTION becuase it's covered by INPUT_ACTION?

No, I said that to specify that adding a new line does dispatch an INPUT_ACTION event, which only happens when the text is multiline. ENTER_ACTION only checks for if enter was pressed regardless of if the text is multiline or not, so its not guaranteed to be caused from a change in the text.

Reply:
sorry for late reply, this week has been a shit show for me, if I understand correctly, then this sounds fine

Reply:
Thinking out loud here: At first I thought an enter event seemed unneeded, but I suppose if the dev wants to, say send a chat message on enter or switch focus to the next field, then it's important that the enter action is handled via the action order rather than `FlxG.keys.justPressed` because that might cause actions to happen out of order

good thinking!

Reply:
Honestly, I only added it here because the original flixel-ui input text had it 😅 But you're right, it does seem useful to keep